### ISSUE TITLE
Razer Ornata V3 Tenkeyless

### ISSUE THREAD

**Issue by laiyiking** [stickoking/razer-macos#52]
Can you add support for Razer Ornata V3 Tenkeyless?
Thanks

**Comment by ericsmsgs** [comment]
I second this request

**Comment by kdumovic** [comment]
Yes please! I assumed that with support of the Ornata Chroma it would support the Tenkeyless but alas, no. When I open the app it says "Please select a device to configure" and that's it.
